packages/common/src/adapters/access.ts

```diff
@@ -34,3 +39,27 @@ export const accessConditionsFromSDK = (
     throw new Error(`Unsupported access gate type: ${input}`)
   }
 }
+export const accessConditionsToSDK = (
+  input: AccessConditions
+): TrackMetadata['downloadConditions'] => {
+  if ('follow_user_id' in input) {
+    return {
+      followUserId: encodeHashId(input.follow_user_id)
```

Comment:
Worth considering using the `Id` and `OptionalId` zod parsers for this. I think the only reason I didn't do it for the other adapters is that the decode versions don't exist yet.

Reply:
Oh yup, forgot about that!